indico/modules/events/tracks/client/js/index.js

```diff
@@ -8,6 +8,33 @@
 (function(global) {
   'use strict';
 
+  function updateSorting() {
+    const sortedList = $('#track-list-container')
```

Comment:
`const container = $('#track-list-container');` and use that one here and below to get the URL